.github/workflows/shellcheck.yaml

```diff
@@ -24,13 +24,10 @@ jobs:
     name: Shellcheck
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-    - name: Run ShellCheck
-      uses: ludeeus/action-shellcheck@00b27aa7cb85167568cb48a3838b75f4265f2bca # master
-      with:
-        severity: warning
-        check_together: 'yes'
-        disable_matcher: false
-        additional_files: build/reset build/sed-in-place
-        ignore: olm
-        format: gcc
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Was this yaml spacing change needed? Or you're just fixing the spacing while touching the file? I was thinking enabling the linting of the workflows directory would be in a separate PR, but I'm fine including this here.

Reply:
@travisn :
I don't see yet how the above diff is just a whitespace change.

The first three line removals are the first part of the refactor. this is part of what is added to the `shellcheck` make target.

I have not intended to include any yaml fixes in this PR

Reply:
I was just commenting on the indentation, but you're right, everything this pretty changed, so I take my comment back. :)